release.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
```

Comment:
Don't need this file, just run bump2version and make dist

Reply:
does bump2version handle both the pypi and the npm package versions?

Reply:
All I want is a one-click release doodad that uses the version in `jupyterfs/_version.py` as the sole source of truth and makes both a pypi and npmjs release. `release.py` is a cleaned-up version of the release script that I wrote a while back for jupyterlab-git that accomplishes this.

If you feel strongly about not including it, we could probably build out the same functionality in the Makefile. What are the relevant bump2version commands?

Reply:
yep this is what the makefile command does already